tasks/src/feature-extraction/about.md

```diff
@@ -0,0 +1,60 @@
+## Use Cases
+
+Feature extraction is the task of extracting meaningful data from the given dataset. It is instrumental in reducing redundant/garbage data. Feature extraction is one of the most trivial steps in machine learning and is used heavily in NLP. In simple terms, Feature Extraction is transforming textual data into numerical data.
+ 
+## Task Variants 
+The task of feature extraction can be divided into three different variants:
+
+### Countvectorizer 
```

Comment:
These are not task variants but rather techniques used to vectorize a given text. Maybe you can reframe them differently 🙂